vendors/tpl-systemes/drivers/birdy-slim-iot/metadata.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "birdy-slim-iot-driver",
+  "version": "1.0.0",
+  "main": "index.js",
+  "description": "Birdy Slim IoT driver",
+  "codecId": "0001",
+  "sourceUrl": "https://github.com/actility/device-catalog/tree/main/template/tpl-systemes/drivers/birdy-slim-iot-driver",
+  "vendorId": "000E",
+  "specVersion": "1.0.0",
+  "codec": {
+    "encodeDownlink": true,
+    "decodeDownlink": true,
```

```suggestion
    "decodeDownlink": false,
```
No decodeDownlink defined in the driver